src/scripts/popup.js

```diff
@@ -74,15 +74,20 @@ document.addEventListener('DOMContentLoaded', () => {
  * @param {HTMLElement} [huntTimerField] The div element corresponding to the horn countdown timer.
  */
 function updateHuntTimerField(tab, huntTimerField) {
+    const hornImage = '<img src="images/horn.png" class="horn-img">';
```

Comment:
this loads horn image into memory every second, vs before we only loaded it when the response was ready. this has performance impact, no?

Reply:
Updated. Good catch.